src/main/java/io/github/antishake/AntiShake.java

```diff
@@ -41,35 +39,37 @@
 
   // To load the config.properties when the class is loaded
   static {
-    InputStream is = null;
-    try {
-      properties = new Properties();
-      is = ClassLoader.class.getResourceAsStream("/config.properties");
-      properties.load(is);
-      loadProperties();
-    } catch (FileNotFoundException e) {
-      logger.error(e);
-    } catch (IOException e) {
-      logger.error(e);
-    }
```

Comment:
Somehow, the static block didn't show up when library was imported in Android app. That's the only reason I moved it.